hoodie-bench/src/main/java/com/uber/hoodie/integrationsuite/helpers/DFSTestSuitePathSelector.java

```diff
@@ -0,0 +1,74 @@
+package com.uber.hoodie.integrationsuite.helpers;
+
+import com.uber.hoodie.common.util.TypedProperties;
+import com.uber.hoodie.common.util.collection.ImmutablePair;
+import com.uber.hoodie.common.util.collection.Pair;
+import com.uber.hoodie.exception.HoodieIOException;
+import com.uber.hoodie.utilities.sources.helpers.DFSPathSelector;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.FileStatus;
+import org.apache.hadoop.fs.LocatedFileStatus;
+import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.fs.RemoteIterator;
+
+/**
+ * A custom dfs path selector used only for the hudi test suite. To be used only if workload is not run inline.
+ */
+public class DFSTestSuitePathSelector extends DFSPathSelector {
```

Comment:
From usability perspective, would greatly help if we a bunch of ready to run test showing the power of hoodie-bench in testing. Maybe you already planned for this.  Let me know your thoughts ?

Reply:
Yes, in the following PR I plan to write up atleast 1 test to cover all possible cases supported by hoodie-bench

Reply:
I feel we should have one end-end test before we land this.